utils/constants/vm_ids.go

```diff
@@ -8,5 +8,6 @@ import "github.com/ava-labs/avalanchego/ids"
 var (
 	PlatformVMID = ids.ID{'p', 'l', 'a', 't', 'f', 'o', 'r', 'm', 'v', 'm'}
 	AVMID        = ids.ID{'a', 'v', 'm'}
-	EVMID        = ids.ID{'e', 'v', 'm'}
+	CorethID     = ids.ID{'e', 'v', 'm'}
+	SubnetEVMID  = ids.ID{'s', 'u', 'b', 'n', 'e', 't', 'e', 'v', 'm'}
```

Comment:
This was added so that subnet-evm metrics don't have the vmID in them. I'm okay with removing it... But it seemed nice to have... I'm not willing to import subnet-evm to grab this ID directly though.